test/gtest/ucs/test_time.cc

```diff
@@ -22,6 +23,8 @@ UCS_TEST_F(test_time, time_calc) {
     EXPECT_NEAR(value * 1000, ucs_time_to_nsec(ucs_time_from_usec(value)), 1.0);
 }
 
+/* This test is only useful when used with high-precision timers */
+#if HAVE_HW_TIMER
```

Comment:
maybe instead we can skip the test in runtime if ticks per second is less than 1e8?
and then define the macro for ARM only?